<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateComsolFile</class>
 <widget class="QFrame" name="GenerateComsolFile">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate COMSOL file</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Modal simulation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="modalSimulationFLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="leftMargin">
      <number>18</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbMeshSizes">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Extremely fine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra fine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Finer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coarse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coarser</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra coarse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extremely coarse</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lSolveProblem">
       <property name="text">
        <string>Solve problem</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="cbSolveProblem">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lMeshSizes">
       <property name="text">
        <string>Mesh sizes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bGenerate">
       <property name="text">
        <string>Generate file...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
